src/components/AusMomentContributionsBanner.stories.tsx

```diff
@@ -1,26 +1,47 @@
 import React, { ReactElement } from 'react';
 import { AusMomentContributionsBanner } from './modules/contributionsBanners/AusMomentContributionsBanner';
-import { withKnobs, boolean, number } from '@storybook/addon-knobs';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
 import { StorybookWrapper } from '../utils/StorybookWrapper';
+import { TickerCountType, TickerEndType } from '../lib/variants';
 
 export default {
     component: AusMomentContributionsBanner,
     title: 'Components/AusMomentContributionsBanner',
     decorators: [withKnobs],
 };
 
+const tracking = {
+    ophanPageId: 'kbluzw2csbf83eabedel',
+    ophanComponentId: 'ACQUISITIONS_ENGAGEMENT_BANNER',
+    platformId: 'GUARDIAN_WEB',
+    clientName: 'dcr',
+    referrerUrl: 'http://localhost:3030/Article',
+    abTestName: 'AusMomentContributionsBanner',
+    abTestVariant: 'control',
+    campaignCode: 'AusMomentContributionsBanner_control',
+};
```

Comment:
Do we still want these values for pageId and referrerUrl ?

Reply:
this is the storybook configuration, so it's only for local development

Reply:
ah that makes sense!